src/python_testing/TC_EEVSE_2_3.py

```diff
@@ -0,0 +1,445 @@
+
+        self.step("16d")
+        await self.check_evse_attribute("NextChargeTargetSoC", NullValue)
+
+        self.step("17")
+        get_targets_response = await self.send_get_targets_command()
+        self.log_get_targets_response(get_targets_response)
+        asserts.assert_equal(get_targets_response, empty_targets_response,
+                             f"Unexpected 'GetTargets' response value - expected {empty_targets_response}, was {get_targets_response}")
+
+        self.step("18")
+        daily_targets_step_18 = [Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+            targetTimeMinutesPastMidnight=60, addedEnergy=25000000)]
+        targets_step_18 = [Clusters.EnergyEvse.Structs.ChargingTargetScheduleStruct(dayOfWeekForSequence=0x1, chargingTargets=daily_targets_step_18),
+                           Clusters.EnergyEvse.Structs.ChargingTargetScheduleStruct(dayOfWeekForSequence=0x1, chargingTargets=daily_targets_step_18)]
+        await self.send_set_targets_command(chargingTargetSchedules=targets_step_18, expected_status=Status.ConstraintError)
+
+        self.step("19")
+        daily_targets_step_19 = [
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=60, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=180, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=300, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=420, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=540, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=660, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=780, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=900, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=1020, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=1140, addedEnergy=25000000),
+            Clusters.EnergyEvse.Structs.ChargingTargetStruct(
+                targetTimeMinutesPastMidnight=1260, addedEnergy=25000000),
+        ]
+
+        targets_step_19 = [Clusters.EnergyEvse.Structs.ChargingTargetScheduleStruct(
+            dayOfWeekForSequence=0x40, chargingTargets=daily_targets_step_19)]
+        await self.send_set_targets_command(chargingTargetSchedules=targets_step_19, expected_status=Status.ResourceExhausted)
+
+        self.step("20")
```

Comment:
We should add a TestEvent clear for the ToU mode as well.

Reply:
Will be implemented when issue https://github.com/project-chip/connectedhomeip/issues/34249 is addressed.